Document new Airzone cloud integration (#27456)
home-assistant/home-assistant.github.io: source/_integrations/airzone_cloud.markdown

---
title: Airzone Cloud
description: Instructions on how to integrate Airzone Cloud within Home Assistant.
ha_release: 2023.6
ha_category:
  - Sensor
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_domain: airzone_cloud
ha_platforms:
  - sensor
ha_codeowners:
  - '@Noltari'
ha_integration_type: integration
---

This integration interacts with the Cloud API of [Airzone HVAC zoning systems](https://www.airzone.es/en/).

> A typical Airzone device has a *parent zone* (Master Thermostat) per HVAC system, which is the only zone where the HVAC mode can be changed. The rest are *child zones* which can only enable or disable the HVAC and adjust the desired temperature on that specific zone.

Reviewer: I'm not familiar with Airzone and therefore struggled a bit when reading this. Some questions:
1. What are these zones? Could you maybe write an introductory sentence on what a zone is in this context? Is it something like my kitchen or living room?
1. Instead of using the master/slave terminology (master thermostat), would any of these [alternative terms](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/master-slave) work? 
1. What is a HVAC mode? What are possible values?

Author: > I'm not familiar with Airzone and therefore struggled a bit when reading this. Some questions:
> 
> 1. What are these zones? Could you maybe write an introductory sentence on what a zone is in this context? Is it something like my kitchen or living room?
> 2. Instead of using the master/slave terminology (master thermostat), would any of these [alternative terms](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/master-slave) work?
> 3. What is a HVAC mode? What are possible values?

1. Zones means rooms with Airzone Thermostats in Airzone terminology. There's more information in these links (from the official website linked on the top):
- https://www.airzone.es/en/comfort/
- https://doc.airzone.es/AZEU/Marketing/The_New_Era_Of_Air_Conditioning_Airzone.pdf
2. Airzone uses Master to refer to the parent thermostat, so I prefer to keep that single "parent zone" == "Master Thermostat" relation to make it more clear to the users reading it.
3. See "hvac_mode" in https://www.home-assistant.io/integrations/climate/

> A typical Airzone device has a *parent zone* (Master Thermostat) per HVAC system, which is the only zone where the HVAC mode can be changed. The rest are *child zones* which can only enable or disable the HVAC and adjust the desired temperature on that specific zone.

Reviewer: ```suggestion
A typical Airzone HVAC system consists of a parent device (called *master* in Airzone terminology) and child devices (called *zones* in Airzone terminology). The [HVAC mode](https://www.home-assistant.io/integrations/climate/#service-climateset_hvac_mode) can only be changed on the parent device. On child devices (*zones*), you can only enable or disable the HVAC and adjust the desired temperature for that specific device.
```
Thank you for providing additional information. I took the liberty of rephrasing this section of the text a bit. This is what I understood from the Airzone documentation. Let me know what you think.

> A typical Airzone HVAC system consists of a parent device (called *master zone* in Airzone terminology) and child devices (called *slave zones* in Airzone terminology). The [HVAC mode](https://www.home-assistant.io/integrations/climate/#service-climateset_hvac_mode) can only be changed on the parent device. On child devices you can only enable or disable the HVAC and adjust the desired temperature for that specific device.

Reviewer: ```suggestion
A typical Airzone HVAC system consists of a parent device (called *master zone* in Airzone terminology) and child devices (called *slave zones* in Airzone terminology). The [HVAC mode](https://www.home-assistant.io/integrations/climate/#service-climateset_hvac_mode) can only be changed on the parent device. On child devices, you can only enable or disable the HVAC and adjust the desired temperature for that specific device.
```

> Note that multiple HVAC systems can be connected to the same Airzone WebServer. In this case, there will be a *parent zone* per HVAC system and there may also be *child zones* for each HVAC system.

Reviewer: ```suggestion
Note that multiple HVAC systems can be connected to the same Airzone web server. In this case, there will be one *parent zone* per HVAC system and there may also be *child zones* for each HVAC system.
```
Unless web server is part of a product name, there is no need to capitalize it. We try to use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization) for common nouns.

{% include integrations/config_flow.md %}

{% configuration_basic %}
Username:
  description: "Cloud API username"
Password:

>   description: "Cloud API Password"

Reviewer: ```suggestion
  description: "Cloud API password"
```

{% endconfiguration_basic %}

## Sensors

> For each Airzone zone (Thermostat), the following *sensors* are created:

Reviewer: ```suggestion
For each Airzone zone (thermostat), the following *sensors* are created:
```

| Condition           | Description                        |
| :------------------ | :--------------------------------- |

> | humidity            | Current zone relative humidity.    |

Reviewer: ```suggestion
| humidity            | Measures the relative humidity in the current zone.     |
```

> | temperature         | Current zone temperature.          |

Reviewer: ```suggestion
| temperature         | Measures the temperature  in the current zone.          |
```
